mmv1/third_party/terraform/resources/resource_compute_instance.go.erb

```diff
@@ -351,6 +351,54 @@ func resourceComputeInstance() *schema.Resource {
 								},
 							},
 						},
+
+						"stack_type": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							ValidateFunc: validation.StringInSlice([]string{"IPV4_ONLY", "IPV4_IPV6", ""}, false),
+							Description: `The stack type for this network interface to identify whether the IPv6 feature is enabled or not. If not specified, IPV4_ONLY will be used.`,
+							Default:      "IPV4_ONLY",
```

Comment:
Conversely, this one *does* appear to be returned. There's a question of whether it's also returned for old resources created before the field was introduced, and I'm not sure what the answer is. If you happen to have an old GCE instance lying around, that'd be worth checking!

Reply:
i checked, it is _not_ returned on old instances. thanks for pointing this out!